.github/workflows/release.yml

```diff
@@ -0,0 +1,118 @@
+
+      - name: Build static veristat natively for amd64
+        if: matrix.arch == 'amd64'
+        working-directory: 'veristat'
+        run: |
+          EXTRA_CFLAGS=--static \
+              make -j -C src V=1
+          strip src/veristat
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@e81a89b1732b9c48d79cd809d8d81d79c4647a18 # v2.1.0
+        if: matrix.arch == 'arm64'
+        with:
+          platforms: arm64
+
+      # The emulated build leverages Docker and Ubuntu 22.04 container image
+      # distribution to have all the needed arm64 packages.
+      - name: Build static veristat for arm64 with emulation
+        if: matrix.arch == 'arm64'
+        run:  |
+          docker run --platform linux/arm64 --rm -v $(pwd):/build ubuntu:22.04 \
+          bash -c "apt-get update && \
+                   apt-get install -y make pkg-config gcc libelf-dev && \
+                   cd /build/veristat && \
+                   EXTRA_CFLAGS=--static \
+                       make -j -C src V=1 && \
+                   strip src/veristat"
+
+      - name: Test veristat binary
+        working-directory: 'veristat/src'
+        env:
+          ARCH: ${{ env[format('FILE_STRING_ARCH_{0}', matrix.arch)] }}
+        run: |
+          file ./veristat | \
+              tee /dev/stderr | \
+              grep -q "${{ env.ARCH }}"
+          ./veristat --usage | grep -q Usage
+          ldd ./veristat 2>&1 | \
+              tee /dev/stderr | \
+              grep -q 'not a dynamic executable'
+
+      - name: Upload Artifact
+        uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
+        with:
+          name: ${{ format('veristat_{0}', matrix.arch) }}
+          path: veristat/src/veristat
+
+  draft-release:
+    name: Create a draft release
+    runs-on: ubuntu-22.04
+    needs: build
+    permissions:
+      contents: write
+    steps:
+      - name: Download artifacts from build
+        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
+
+      - name: Rename binaries and compress
+        run: |
+          archive_amd64="veristat-${{ github.ref_name }}-amd64.tar.gz"
+          archive_arm64="veristat-${{ github.ref_name }}-arm64.tar.gz"
+          tar -C veristat_amd64 -I 'gzip -9' -cvf "${archive_amd64}" veristat
+          tar -C veristat_arm64 -I 'gzip -9' -cvf "${archive_arm64}" veristat
+          sha256sum "${archive_amd64}" > "${archive_amd64}.sha256sum"
+          sha256sum "${archive_arm64}" > "${archive_arm64}.sha256sum"
+
+      - name: Checkout veristat and libbpf code
+        uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # v3.5.0
+        with:
+          submodules: recursive
+          path: 'veristat'
+
+      - name: Package source code including submodules
+        run: |
+          tar -I 'gzip -9' --exclude-vcs \
+              -cvf "veristat-libbpf-${{ github.ref_name }}-sources.tar.gz" veristat
```

Comment:
can you please drop `-libbpf` part? To emphasize that it's sources of everything need by veristat we can do "all-sources" prefix or something like that.